app/views/users/_new_game_form.html.erb

```diff
@@ -6,10 +6,11 @@
   <div class="form-group" id="new-game">
     <input class="form-control hidden" type="text" id="current-user" value="<%= current_user.id %>">
     <input class="form-control hidden" type="text" id="current-role" value="<%= current_user.role %>">
-    <input class="form-control" type="text" id="game-name" value='Name'>
-    <input class="form-control" type="url" id="game-heroku" value='Game Link: Must be the full url'>
-    <input class="form-control" type="url" id="game-github" value = 'Github URL: Must be the full url'>
-    <input class="form-control" type="url" id="game-screenshot" value= "Gif: Must be the full url">
+    <input class="form-control" type="text" id="game-name" placeholder='Name'>
+    <input class="form-control" type="url" id="game-heroku" placeholder='Game Link: Must be the full url'>
+    <input class="form-control" type="url" id="game-github" placeholder = 'Github URL: Must be the full url'>
```

Comment:
some html styling: use double quotes, and keep equals signs without spaces on either side